File: pinyougou_parent/pinyougou_portal_web/src/main/webapp/js/service/promotionService.js
//服务层
app.service('promotionService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findPromotionListByStatus=function(){
		return $http.get('../promotion/findPromotionListByStatus');		
	}
	
	this.updatePromotionStatus=function(promotion){
		return $http.post('../promotion/updatePromotionStatus',promotion);
	}
	
	this.deletePromotionById=function(id){
		return $http.post('../promotion/deletePromotionById/'+id);
	}
	
});

File: pinyougou_parent/pinyougou_content_interface/src/main/java/com/pinyougou/content/service/PromotionService.java
package com.pinyougou.content.service;

import java.util.List;

import com.pinyougou.pojo.TbPromotion;

public interface PromotionService {

	List<TbPromotion> findPromotionListByStatus();

	void updatePromotionStatus(TbPromotion promotion);

	void deletePromotionById(Long id);



}
